tests/test_utils.py

```diff
@@ -53,31 +54,54 @@
 
 logger = logging.getLogger(__name__)
 
-BASE_TMP_DIR = pathlib.Path('tests/__tmp_path').resolve()
-YAML_TEST_DATA_PATH = pathlib.Path('tests/data/yaml/').resolve()
-JSON_TEST_DATA_PATH = pathlib.Path('tests/data/json/').resolve()
-ENV_TEST_DATA_PATH = pathlib.Path('tests/data/env/').resolve()
-JSON_NIST_DATA_PATH = pathlib.Path('nist-content/nist.gov/SP800-53/rev5/json/').resolve()
+TEST_DIR = pathlib.Path(__file__).parent.resolve()
+BASE_TMP_DIR = pathlib.Path(TEST_DIR / '__tmp_path').resolve()
+YAML_TEST_DATA_PATH = pathlib.Path(TEST_DIR / 'data/yaml/').resolve()
+JSON_TEST_DATA_PATH = pathlib.Path(TEST_DIR / 'data/json/').resolve()
+ENV_TEST_DATA_PATH = pathlib.Path(TEST_DIR / 'data/env/').resolve()
+JSON_NIST_DATA_PATH = pathlib.Path(TEST_DIR / '../nist-content/nist.gov/SP800-53/rev5/json/').resolve()
 JSON_NIST_CATALOG_NAME = 'NIST_SP-800-53_rev5_catalog.json'
 JSON_NIST_PROFILE_NAME = 'NIST_SP-800-53_rev5_MODERATE-baseline_profile.json'
-JSON_NIST_REV_4_DATA_PATH = pathlib.Path('nist-content/nist.gov/SP800-53/rev4/json/').resolve()
+JSON_NIST_REV_4_DATA_PATH = pathlib.Path(TEST_DIR / '../nist-content/nist.gov/SP800-53/rev4/json/').resolve()
 JSON_NIST_REV_4_CATALOG_NAME = 'NIST_SP-800-53_rev4_catalog.json'
 JSON_NIST_REV_5_CATALOG_NAME = 'nist-rev5-catalog-full.json'
 JSON_NIST_REV_4_PROFILE_NAME = 'NIST_SP-800-53_rev4_MODERATE-baseline_profile.json'
 SIMPLIFIED_NIST_CATALOG_NAME = 'simplified_nist_catalog.json'
 SIMPLIFIED_NIST_PROFILE_NAME = 'simplified_nist_profile.json'
-TASK_XLSX_OUTPUT_PATH = pathlib.Path('tests/data/tasks/xlsx/output').resolve()
+TASK_XLSX_OUTPUT_PATH = pathlib.Path(TEST_DIR / 'data/tasks/xlsx/output').resolve()
 
 CATALOGS_DIR = 'catalogs'
 PROFILES_DIR = 'profiles'
 COMPONENT_DEF_DIR = 'component-definitions'
 
-NIST_EXAMPLES = pathlib.Path('nist-content/examples')
+NIST_EXAMPLES = pathlib.Path(TEST_DIR / '../nist-content/examples')
 NIST_SAMPLE_CD_JSON = NIST_EXAMPLES / 'component-definition' / 'json' / 'example-component.json'
 
 NEW_MODEL_AGE_SECONDS = 100
 
 
+def set_cwd_unsafe(cwd: pathlib.Path = TEST_DIR):
+    """Set the current working directory to the test directory.
+
+    Not safe for concurrent tests which may change directory.
+    """
+
+    def decorator(f):
+
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            original_cwd = os.getcwd()
+            os.chdir(cwd)
+            try:
+                f(*args, **kwargs)
+            finally:
+                os.chdir(original_cwd)
+
+        return wrapper
+
+    return decorator
+
+
```

Comment:
`Not safe for concurrent tests which may change directory.` This is somewhat concerning. Running `make test` causes a set of parallel chunks of tests to be run. It is good that all the tests passed for this PR, but was that luck or not? Thx.

Reply:
I believe that this is practically a no-op in the case of `make test` because the decorator `cd`s to the root directory and `make test` only works in the root directory anyway, so the decorator starts at the test root, then cds to the test root, then cds back to the test root after the test runs.
It's good that this unsafe workaround triggers raises a mental flag because it is only needed on tests that have an implicit dependency on paths relative to the test root directory

Reply:
OK. I suggest adding similar comments in the code itself.